### ISSUE TITLE
About prunning channels

### ISSUE THREAD

**Issue by Ariel666** [mengrang/Slimming-pytorch#1]
Have you completed this project? I haven't seen the pruning channels codes....
Do you plan to actually prune the channels from the network?

**Comment by mengrang** [comment]
I don't think you have understood my codes.Did you see the real_prune.py and the new_resnet.py?Actually, I do the real pruning, but just haven't fine tuned.

**Comment by Ariel666** [comment]
Sorry, I didn't see the saving model codes after pruning.   Can you show me that?
Besides, how can I use new_resnet? Did I use real_prune to prune channels and use new_resnet to construct a new network?

**Comment by mengrang** [comment]
Sorry for replying you later, First,  I run the train.py to prune while training,then use the real_prune.py to remove the unimportant channels, finnally  I load the pruned net weights to new_resnet

**Comment by Fly-Pluche** [comment]
Could you reply me why I update the slim_bnw_dict to the model_dict  and save the model_dict but find that the size of model as big as the same.
